cards/src/main/resources/cards/custom/group3/bug/verdant_dreams/minion_the_glutton.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "The Glutton",
+  "baseManaCost": 3,
+  "type": "MINION",
+  "heroClass": "AMBER",
+  "baseAttack": 3,
+  "baseHp": 3,
+  "rarity": "LEGENDARY",
+  "race": "BEAST",
+  "description": "At the end of your turn, destroy a random minion with less Attack and gain +1/+1.",
```

Comment:
this should be a conditional spell, i'm pretty sure if there's no such minion it will still buff itself. i don't think that's the intended effect. to be clear, the intended effect is, if there's a minion to destroy, destroy it and buff me.

Reply:
tested and fixed